### PR TITLE
UNFILED: try to fix failing int test

### PR DESCRIPTION
**Pull request by jeremy-flieder**

### CODE REVIEW

File: Apps/RaptureIntTests/src/test/java/rapture/lock/LockWorkflowTest.java [hunk 1]
```diff
@@ -562,7 +563,9 @@ public void testBlockingLockWithinWorkflows() {
 		       	lockHandle.setLockName(decisionApi.getContextValue(orderUri+"#0", "lockName"));	
 	       	}
         }
-        Assert.assertTrue(lockApi.releaseLock(lockUri.toString(), lockConfig.getName(), lockHandle));
+        try {
```
Comment (jonathan-major) [inline Apps/RaptureIntTests/src/test/java/rapture/lock/LockWorkflowTest.java]: is a Rapture Exception expected when calling lockApi.releaseLock()? Seems off to me.

Comment (scarabus) [inline Apps/RaptureIntTests/src/test/java/rapture/lock/LockWorkflowTest.java]: Possibly, if you try to release a lock that you don't hold.

Comment (jeremy-flieder) [inline Apps/RaptureIntTests/src/test/java/rapture/lock/LockWorkflowTest.java]: I believe Dave is correct in his description of how releaseLock works. Should i file a ticket to change it to not throw an exception (and also not release lock) ?

Comment (jonathan-major) [inline Apps/RaptureIntTests/src/test/java/rapture/lock/LockWorkflowTest.java]: if the
1.  behavior is documented in the api doc
2. the exception message states "you aren't the owner of the lock.."

 then no need to open a new ticket.

Comment (jeremy-flieder) [inline Apps/RaptureIntTests/src/test/java/rapture/lock/LockWorkflowTest.java]: Created this to address lack of documentation:

https://incapture.atlassian.net/browse/RAP-4355

**Comment by scarabus** [comment]
LockApiImpl simply called the underlying lock handler - If the handler throws an exception then that exception will be propagated. 
ZooKeeperLockHandler throws an exception if it can't find the lock, but catches and logs at a low level any exception thrown by the Zookeeper lock.release() call - whereas MongoLockHandler does not explicitly throw or catch anything.

**Comment by jonathan-major** [comment]
ah a difference in expected behavior @jeremy-flieder can you log and @scarabus fix please?

**Comment by jeremy-flieder** [comment]
https://incapture.atlassian.net/browse/RAP-4356

**Pull request merged**